web/src/settings_invites.ts

```diff
@@ -191,12 +193,14 @@ function do_resend_invite({$row, invite_id}: {$row: JQuery; invite_id: string}):
         blueslip.error("Invite resending canceled due to non-matching fields.");
         ui_report.client_error(
             $t_html({
-                defaultMessage: "Resending encountered an error. Please reload and try again.",
+                defaultMessage: "Error: Could not resend invitation.",
             }),
-            $("#home-error"),
+            $("#resend_invite_modal #dialog_error"),
         );
+        return;
     }
 
+    dialog_widget.close();
```

Comment:
I don't get why is this needed.

Reply:
I used `dialog_widget`(instead of `confirm_dialog`), which does not close automatically since `close_on_submit` is not passed as `true`. Therefore, I had to call `dialog_widget.close();` to close the modal when no errors are encountered.

Thank you for the review. Please let me know if any further adjustments are needed.

Reply:
Ohh. Yes, this seems fine for now. But we should just move this to call to `success` function of the API call, if we decide to show to error returned by server in modal as well.